js/compareJIT.py

```diff
@@ -28,6 +29,7 @@
 from FTB.ProgramConfiguration import ProgramConfiguration
 
 
+gOptions = ""
```

Comment:
This can be restructured to eliminate the global if you wish. In my private interesting script, I do something like this:

    class MyInteresting(object):
    
        def __init__(self, interesting_script=False):
            if interesting_script:
                global init, interesting, cleanup # pylint: disable=global-variable-undefined, invalid-name
                init = self.init
                interesting = self.interesting
                cleanup = self.cleanup
            ...
    
        def init(self, args):
            ...
    
        def interesting(self, args, temp_prefix):
            ...
    
        def cleanup(self, args):
            ...
    
    if __name__ == "__main__":
        main()
    else:
        # create an anonymous instance with methods assigned as globals where lithium expects them
        MyInteresting(True)

Then `gOptions` can become `self.options`.

This way also allows using it as a class with the `Lithium` class programmatically in the future, instead of using `lithium.py` as a script.